src/typeset_constructs/typeset_settings.h

```diff
@@ -0,0 +1,50 @@
+#ifndef TYPESET_SETTINGS_H
+#define TYPESET_SETTINGS_H
+
+#include "typeset_construct.h"
+#include "forscape_dynamic_settings.h"
+
+#ifndef FORSCAPE_TYPESET_HEADLESS
+#include <typeset_command_change_settings.h>
+#include <typeset_controller.h>
+#include <typeset_model.h>
+#include <typeset_settings_dialog.h>
+#endif
+
+//DO THIS: should reveal settings on hover
+//DO THIS: should open on click or double-click, maybe have a button for expand/condense
```

Comment:
Seriously, implement the standard and expected interactions. Make sure the settings look good when highlighted. We want polish.